ci/nightly-build.sh

```diff
@@ -30,5 +30,5 @@ ${MVN} clean package ${MVN_MIRROR}  \
   -Psource-javadoc \
   -DCPP_PARALLEL_LEVEL=${PARALLEL_LEVEL} \
   -Dlibcudf.build.configure=true \
-  -DUSE_GDS=${USE_GDS} -Dtest=*,!CuFileTest,!CudaFatalTest \
+  -DUSE_GDS=${USE_GDS} -Dtest=*,!CuFileTest,!CudaFatalTest,!ColumnViewNonEmptyNullsTest \
```

Comment:
Try to undo this and check if it gets rid of the exclude override for ColumnViewNonEmptyNullsTest 

```suggestion
  -DUSE_GDS=${USE_GDS} -Dtest=*,!CuFileTest,!CudaFatalTest \
```

and check if you can manually run with the flags before your PR

Reply:
OK, I will try that next.